vscode/package.json

```diff
@@ -274,8 +274,18 @@
             "auto",
             "rubocop",
             "syntax_tree",
+            "standard",
+            "rubyfmt",
             "none"
           ],
+          "enumDescriptions": [
+            "Auto",
+            "RuboCop",
+            "Syntax Tree",
+            "Standard (need v0.37 or newer of the standard gem)",
```

Comment:
We'll co-ordinate with `@searls` to determine the release version for the work being done in https://github.com/standardrb/standard/pull/630.